Cargo.toml

```diff
@@ -17,6 +17,7 @@ bitflags = "~1"
 crossbeam-epoch = "~0.7"
 # TODO: Consider what to do with the union feature. Why is it still requiring nightly?
 smallvec = "~0.6"
+rayon = "~1"
```

Comment:
I'd prefer to have rayon as an optional dependency only, so people who don't need it don't need to bring in quite a big dependency tree. Would you mind adding the (non-default) feature flag? https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section

Reply:
Not sure what you want to call it, I figured you may put more than `rayon` in that particular feature someday so I called it `parallel`.

Reply:
As described at the comment, I'd prefer for it to actually be `rayon`. It's more granular and specific.